scoop-dagger/src/main/java/com/lyft/scoop/dagger/DaggerScreenScooper.java

```diff
@@ -28,4 +31,14 @@ protected Scoop addServices(Scoop.Builder scoopBuilder, Screen screen, Scoop par
         return scoopBuilder
                 .service(DaggerInjector.SERVICE_NAME, screenInjector).build();
     }
+
+    public static Scoop extend(final Scoop parentScoop, final Object... modules) {
```

Comment:
@itspbj Let's move extend logic to DaggerInjector. ScreenScooper should not be exposed for usages in views or controllers.